applications/eigen_solvers_application/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+//  KRATOS _______
+//        / ____(_)___ ____  ____
+//       / __/ / / __ `/ _ \/ __ \
+//      / /___/ / /_/ /  __/ / / /
+//     /_____/_/\__, /\___/_/ /_/ SolversApplication
+//             /____/
+//
+//  Author: Thomas Oberbichler
+*/
+
+// System includes
+
+// External includes
+#include <boost/python.hpp>
+
+// Project includes
+#include "includes/define.h"
+#include "processes/process.h"
+#include "custom_python/add_custom_utilities_to_python.h"
+
+#include "spaces/ublas_space.h"
+
+namespace Kratos
+{
+
+namespace Python
+{
+
+void AddCustomUtilitiesToPython()
```

Comment:
I think you can remove the files `add_custom_utilities_to_python.h` and `add_custom_utilities_to_python.cpp`, they are not doing anything

Reply:
They should be used for the upcoming eigenvalue algorithms